src/library_wasmfs.js

```diff
@@ -139,6 +139,19 @@ mergeInto(LibraryManager.library, {
       });
     },
     // TODO: mkdirTree
+    // Creates a whole directory tree chain if it doesn't yet exist
+    mkdirTree: (path, mode) => {
+        var dirs = path.split('/');
+        var d = '';
+        for (var i = 0; i < dirs.length; ++i) {
+          if (!dirs[i]) continue;
+          d += '/' + dirs[i];
+          return withStackSave(() => {
```

Comment:
This return will exit from the entire function, during the first iteration in the loop, which is not what we want here I think?